simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/it/ChangeEvseAvailabilityIntegrationTest.java

```diff
@@ -0,0 +1,61 @@
+package com.evbox.everon.ocpp.simulator.it;
+
+import com.evbox.everon.ocpp.simulator.mock.StationSimulatorSetUp;
+import com.evbox.everon.ocpp.simulator.station.Station;
+import com.evbox.everon.ocpp.simulator.station.evse.EvseStatus;
+import com.evbox.everon.ocpp.v20.message.station.ChangeAvailabilityRequest;
+import org.junit.jupiter.api.Test;
+
+import static com.evbox.everon.ocpp.simulator.mock.ExpectedCount.twice;
+import static com.evbox.everon.ocpp.simulator.mock.ExpectedRequests.bootNotificationRequest;
+import static com.evbox.everon.ocpp.simulator.mock.ExpectedRequests.statusNotificationRequest;
+import static com.evbox.everon.ocpp.simulator.mock.ExpectedResponses.bootNotificationResponse;
+import static com.evbox.everon.ocpp.simulator.mock.ExpectedResponses.statusNotificationResponse;
+import static com.evbox.everon.ocpp.simulator.support.JsonMessageTypeFactory.createCall;
+import static com.evbox.everon.ocpp.simulator.support.StationConstants.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class ChangeEvseAvailabilityIntegrationTest extends StationSimulatorSetUp {
+
+    @Test
+    void shouldChangeEvseStatusToUnavailable() {
+
+        ocppMockServer
+                .when(bootNotificationRequest())
+                .thenReturn(bootNotificationResponse());
+
+        ocppMockServer
+                .when(statusNotificationRequest(), twice())
+                .thenReturn(statusNotificationResponse());
+
+        stationSimulatorRunner.run();
+
+        ocppMockServer.waitUntilConnected();
+
+        ocppServerClient.findSender(STATION_ID).sendMessage(createChangeAvailabilityRequest());
```

Comment:
ocppServerClient is a station itself, which is not the case here I believe

Reply:
Its not a station. Its a server-client that sends a message. So message comes from the server.

I dont have an instance of ocppServer, only ocppMockServer and ocppServerClient. 

Naming could be better I agree. How about `ocppServerClient.findStation(STATION_ID).sendMessage(...)`

The reason is OOP. Single responsibility - find and then send.

Reply:
I mean from perspective of OCPP ecosystem 'ocppServerClient' term means station. I'd call it adapter or wrapper then. But still would prefer not to have separate class that allows communication with mock.

Reply:
Its not a station. Its a client that sends a message via websocketchannel. Where does it say that its a station?
And its not an adapter or decorator if your refer GOF patterns. Its a client. About splitting into classes please read below